lib/elixir/lib/inspect/algebra.ex

```diff
@@ -44,19 +44,23 @@ defmodule Inspect.Opts do
 
   """
 
+  # TODO: Deprecase in v1.5 option "char_lists" key
   defstruct structs: true,
             binaries: :infer,
+            charlists: :infer,
             char_lists: :infer,
             limit: 50,
             width: 80,
             base: :decimal,
             pretty: false,
             safe: true
 
+  # TODO: Deprecase in v1.5 option ":as_char_lists" and "char_lists" key.
   @type t :: %__MODULE__{
                structs: boolean,
                binaries: :infer | :as_binaries | :as_strings,
-               char_lists: :infer | :as_lists | :as_char_lists,
+               charlists: :infer | :as_lists | :as_charlists | :as_char_lists,
+               char_lists: :infer | :as_lists | :as_charlists | :as_char_lists,
```

Comment:
I don't know if this is the right way, or if there is a way to hide :char_lists from the user documentation.
On a side note, should we do instead:
```elixir
charlists: :infer | :as_lists | :as_charlists,
char_lists: :infer | :as_lists | :as_charlists,
```
So if user switch to :charlists, we force him to use :as_charlists